Vic2ToHoI4/Data_Files/blankMod/output/decisions/_debug_decisions.txt

```diff
@@ -96,10 +96,11 @@ debug_decisions = {
 		complete_effect = {
 			random_operative = {
 				harm_operative_leader = 1
-				operative_leader_event = {
-					id = lar_operative_event.4
-					set_from_from = URG
-				}
+			#Some easter egg from Vanilla HoI4 - can be removed safely
+			#	operative_leader_event = {
```

Comment:
You know the routine: delete instead of commenting out.

Reply:
Done